src/core/include/ngraph/specialize_function.hpp

```diff
@@ -14,11 +14,17 @@
 #    endif
 #endif
 
-#include "ngraph/function.hpp"
+#include <memory>
+
+#include "ngraph/deprecated.hpp"
+#include "ngraph/ngraph_visibility.hpp"
+#include "openvino/core/model.hpp"
+#include "openvino/core/shape.hpp"
+#include "openvino/core/type.hpp"
 
 namespace ngraph {
```

Comment:
What about `ngraph` namespace?

Reply:
This header is removed in scope of https://jira.devtools.intel.com/browse/CVS-130656
I just added these includes to fix build